src/components/dashboard/dashboardv1.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from "react";
+import { axiosWithAuth } from "../../utils/axiosWithAuth.js";
+import { useHistory } from "react-router-dom";
+
+import axios from "axios";
+
+// local imports
+import "./_Dashboard.scss";
+import MovieCard from "../movies/MovieCard.js";
+// temp import to verify switch statement
+import LoadingScreen from "../layout/LoadingScreen.js";
+
+const Dashboardv1 = () => {
+  let history = useHistory();
+  const [input] = useState({ file: "" });
+  const [ratings, setRatings] = useState({});
+  const [recommendations, setRecommendations] = useState([]);
```

Comment:
These recommendations are currently static. All new users I was making and uploading data for would not return recommendations.